apps/web-remix/app/components/pages/interfaces/form/formInterface.form.tsx

```
@@ -0,0 +1,171 @@
+      payload: {
+        name: string;
+      };
+    }
+  | {
+      type: 'SUBMIT';
+    };
+
+function formReducer(state: State, action: Action) {
+  switch (action.type) {
+    case 'SET_PROPERTY':
+      const errors = { ...state.formErrors };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)